src/ParkChecklist/ParkChecklist.js

```diff
@@ -7,181 +7,222 @@ import page from "../resources/page.png";
-                                        {stateCode}
-                                    </option>
-                                ))}
-                            </select>
-                            <button onClick={handleGoClick}>Go!</button>
-                        </div>
-                    </div>
-                    <h2>{`${totalChecked}/${parks.length} parks visited`}</h2>
-                    <h2>Parks Checklist</h2>
-                    <ul className="checkbox-list">
-                        {currentParks.map((park) => (
-                            <li key={park.id}>
-                                <input
-                                    type="checkbox"
-                                    id={park.id}
-                                    checked={checkedItems.includes(park.id)}
-                                    onChange={() => handleCheckboxChange(park.id)}
-                                />
-                                <label htmlFor={park.id}>{park.fullName}</label>
-                            </li>
-                        ))}
-                    </ul>
-                    <div className="pagination">
-                        <button onClick={() => handlePageChange(1)}>{"<<"}</button>
-                        <button onClick={() => handlePageChange(currentPage - 1)}>
-                            {"<"}
-                        </button>
-                        <span>{`${currentPage}/${Math.ceil(
-                            parks.length / parksPerPage
-                        )}`}</span>
-                        <button onClick={() => handlePageChange(currentPage + 1)}>
-                            {">"}
-                        </button>
-                        <button
-                            onClick={() =>
-                                handlePageChange(Math.ceil(parks.length / parksPerPage))
-                            }
-                        >
-                            {">>"}
-                        </button>
-                    </div>
-                </>
-            ),
-        },
-        {
-            path: `/Parks/${selectedState}`,
-            element: showResults ? (
-                <SelectedStateParks
-                    parks={parks}
-                    selectedState={selectedState}
-                    fetchParkDetails={fetchParkDetails}
+  const routes = useRoutes([
+    {
+      path: "/",
+      element: (
+        <>
+          <div className="below-header-box">
+            <p className="welcome-message">Welcome User!</p>
+            <div className="state-selector">
+              <select onChange={handleStateChange} value={selectedState}>
+                <option value="">Select a State</option>
+                {states.map((stateCode) => (
+                  <option key={stateCode} value={stateCode}>
+                    {stateCode}
+                  </option>
+                ))}
+              </select>
```

Comment:
still cant believe that the adding of the attribute onto all of our data fixed everything